src/common/ceph_crypto.cc

```diff
@@ -83,6 +84,7 @@ void ceph::crypto::shutdown()
   assert(crypto_refs > 0);
   if (--crypto_refs == 0) {
     NSS_ShutdownContext(crypto_context);
+    PR_Cleanup();
```

Comment:
My only concern was related to the same reason why we switched to ```NSS_InitContext``` vs ```NSS_Init``` -- that allows better isolation between libraries and their applications that might also use NSS. Personally, I'd vote to have this skipped for ```CODE_ENVIRONMENT_LIBRARY```